OpenRA.Game/Map/CellLayer.cs

```diff
@@ -45,6 +45,19 @@ public void CopyValuesFrom(CellLayer<T> anotherLayer)
 			Array.Copy(anotherLayer.entries, entries, entries.Length);
 		}
 
+		public static CellLayer<T> CreateInstance(Func<MPos, T> initialCellValueFactory, Size size, TileShape tileShape)
+		{
+			var cellInfoLayer = new CellLayer<T>(tileShape, size);
+			for (var v = 0; v < size.Height; v++)
```

Comment:
Style nit: if an inner block has braces, then outer blocks should also have braces.